docs/proposals/job.md

```diff
@@ -0,0 +1,145 @@
+# Job Controller
+
+## Abstract
+A proposal for implementing a new controller - Job controller - which will be responsible
+for managing pod(s) that require to run-once to a completion, in contrast to what
+ReplicationController currently offers.
+
+Several existing issues and PRs were already created regarding that particular subject:
+* Job Controller [#1624](https://github.com/GoogleCloudPlatform/kubernetes/issues/1624)
+* New Job resource [#7380](https://github.com/GoogleCloudPlatform/kubernetes/pull/7380)
+
+
+## Use Cases
+1. Be able to start of one or several pods tracked as a single entity.
+1. Be able to implement heavy-computation model, eg. MapReduce.
+1. Be able to get the job status.
+1. Be able to limit the execution time for a job.
+1. Be able to create a chain of jobs dependent one on another.
```

Comment:
I'll create a section with future directions at the end of the document, so that it won't be forgotten.